docs/tutorials/update-task-description.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+---
+
+# Update a task description
+
+In this tutorial, you will learn how to use Postman to update a task description. This use case is common, since your users will need to to add task details as their deadlines approach. 
+
+This tutorial takes about 15 minutes to complete. 
+
+## Before you begin
+
+Before starting this tutorial, install Postman and json-server. Then, sync the To-Do service to your  GitHub repository so that you can clone it to your desktop. For a complete set of steps, see Before you start a tutorial. 
```

Comment:
Could you refer to the [Before you start a tutorial](https://uwc2-apidoc.github.io/to-do-service-public/tutorials/before-you-start-a-tutorial.html) topic here?